src/routes/(protected)/settings/your-apps/new/+page.svelte

```diff
@@ -16,21 +38,30 @@
     </label>
     <label for="redirectUri" class="label">
         <span>Redirect URI</span>
-        <input class="input" type="text" name="redirectUri" required bind:value={$form.redirectUri} {...$constraints.redirectUri}  placeholder="Enter redirect URI..." />
+        <div class="input-group input-group-divider grid-cols-[auto_1fr_auto]">
+            <div name="redirectUri" class="input-group-shim" >https://</div>
+            <input  type="text" name="redirectUri" required {...$constraints.redirectUri}  placeholder="Enter redirect URI..." on:change={(e) => urlBoxes(e.target)}/>
```

Comment:
ok this is kinda problematic, the bind:value is required, as for on:change... it can simply be refactored to on:change={handleRedirectUriChange} (yes pass a reference to the function instead of calling it, svelte will call it automatically) where handleRedirectUri can directly update the form's parameter value without passing in anything